### ISSUE TITLE
Feature Request: Support deleted line detection

### ISSUE THREAD

**Issue by agentroadkill** [na-ka-na/ExcelCompare#48]
Detecting deleted lines (by comparing, say two or three lines above and below) would be a great rather than having to manually read the diff output and infer deleted lines.

**Comment by na-ka-na** [comment]
Yes, this has been a long standing TODO. Not just deleted. But say, you inserted one row in the middle, or column etc. It's not trivial to detect these things automagically.
